include/TPP/Passes.td

```diff
@@ -446,4 +446,12 @@ def DefaultPipeline : Pass<"default-pipeline", "ModuleOp"> {
   ];
 }
 
+def ConvertTppToIdentity: Pass<"tpp-to-identity", "func::FuncOp"> {
+  let summary = "Lower TPP transpose to tpp identity";
+  let description = [{
+    Convert tpp transpose operations to tpp identity loops.
```

Comment:
This is not what the pass is doing. The pass is just optimizing the packing algorithm, and most of it can actually go upstream later, because even `memref.copy` can be faster (on a line/tile) than one-by-one exchange.

We're also not replacing all of the `pack`s, so this isn't a conversion pass, but an optimization pass.

I'd call this pass "pack-optimization" and have its description as something like: "Optimize certain `pack` and `unpack` patterns (block-transpose) into faster implementation".

We can have a separate one to lower to XSMM that we keep local.

Reply:
Yepp, see above for now there should be a pass that lower `memref.copy` to `XSMM` identity and `XSMM` norm->vnni. The one for `XSMM` identity already exists and it will need massaging to support norm->vnni.

Reply:
Awesome! Ok, so we don't need TPP identity at all even now. Let's drop the requirement on the pass.

This will make it a lot easier to upstream this pass later on.